### ISSUE TITLE
[Doc] Add components versions

### ISSUE THREAD

**Issue by pylapp** [Orange-OpenSource/ouds-ios#690]
### Documentation Location

In documentation of components we must incldue the versions of the components as defined in the Figma specifications

### Current issue

It is not clear in the documentation: users cannot know if the current implement of a given component is in some version our another version as defined in the Figma specifications

### Expected Correction

Add in docs the components versions
